test/scalability/lib.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import os
+import subprocess
+
+import pandas as pd
+from matplotlib import pyplot as plt
```

Comment:
Nit: Are the aliases necessary? I am not too eager to use short abbreviations unless they are established.

Reply:
They are extremely well established in StackOverflow :-) I think they will eventually rename the modules themselves to the short aliases.

Reply:
Ok, feel free to keep them in this case.